test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/Basic_Runtime.codegen.cs

```diff
@@ -1,4 +1,4 @@
-#pragma checksum "TestFiles/IntegrationTests/CodeGenerationIntegrationTest/Basic.cshtml" "{ff1816ec-aa5e-4d10-87f7-6f4963833460}" "4120ddad9d4353ed260e0585fe71080d78ff8ab3"
+#pragma checksum "TestFiles/IntegrationTests/CodeGenerationIntegrationTest/Basic.cshtml" "{ff1816ec-aa5e-4d10-87f7-6f4963833460}" "6d079dd6c39f39d17a2faff14404b37ab7e8b8fc"
```

Comment:
All these checksum changes are expected. We now use `TestRazorProjectItem` which [includes](https://github.com/aspnet/Razor/blob/dev/test/Microsoft.AspNetCore.Razor.Test.Common/Language/TestRazorProjectItem.cs#L37) UTF8 BOM along with the file content. That resulted in this change.

Reply:
Ok cool beans